django/contrib/admin/options.py

```diff
@@ -2263,7 +2274,8 @@ def history_view(self, request, object_id, extra_context=None):
 
         context = {
             **self.admin_site.each_context(request),
-            "title": _("Change history: %s") % obj,
+            "title": _("Change history: %s")
+            % display_for_value(str(obj), EMPTY_VALUE_STRING),
```

Comment:
There's string casting here and then also inside `display_for_value()`, can you walk me through why we need both?